### ISSUE TITLE
Broadcast to all except sender

### ISSUE THREAD

**Issue by endel** [gamestdio/colyseus#136]
I'd like to allow calling `broadcast(data)`, without sending the message to the sender.

Initially, I'm thinking of:

```typescript
onMessage (client, message) {
  const data = { hello: "world" };

  // 1. passing "options" as second argument
  this.broadcast(data, { except: client });

  // 2. or having a "broadcastExcept" method, 
  //    passing as reference the client that shouldn't receive it
  this.broadcastExcept(data, client);
}
```

Which option would be best? Do you have another suggestion?
Thanks!

**Comment by seiyria** [comment]
I would like this. I think it should accept or coerce into an array of clients though, if possible. I'm sure there are uses I can't think of right now that could benefit from such an approach

**Comment by AnubisCode** [comment]
I think the 2nd option is better.

**Comment by vutrungduc7593** [comment]
1st option is better to extend, example if i want to allow clients instead of except, less params is better (clean code)

**Comment by Wenish** [comment]
first

**Comment by endel** [comment]
Thanks, guys! This has landed on `0.9.0`, too! 

Usage will be:

```
this.broadcast(data, { except: client });
```

If we feel the need for a change, we can do it for `1.0.0`.
